Makefile

```diff
@@ -24,11 +24,14 @@ build: fmt vet
 
 .PHONY: build-container
 build-container: fmt vet test
-	podman build -t ${IMG} .
+	podman build --target=amd64-build --manifest=${IMG} -t ${IMG_REPOSITORY}:amd64-${IMG_TAG} . && \
+	podman build --target=s390x-build --manifest=${IMG} -t ${IMG_REPOSITORY}:s390x-${IMG_TAG} .
 
 .PHONY: push-container
 push-container:
-	podman push ${IMG}
+	podman push ${IMG_REPOSITORY}:amd64-${IMG_TAG} && \
+	podman push ${IMG_REPOSITORY}:s390x-${IMG_TAG} && \
+	podman manifest push ${IMG}
```

Comment:
I'm not familiar with multi-arch images. When the image is pulled, is the arch determined by the system where the podman is running? Can I just call `podman run ${IMG}` ?

Reply:
It can be made this way, but this does not look like it to me. These docs should help: https://docs.podman.io/en/latest/markdown/podman-manifest.1.html#examples

Reply:
@akrejcir, after you have added the manifest and the images to it, you can simply use 'podman run', and it will automatically select the appropriate architecture during the pull.

@0xFelix, you are correct. However, for this to work, the host must have the capability to execute non-native binaries. As I am not familiar with the specifics of your CI workers, I chose a more generic approach.